report-summary-merged-prs

```diff
@@ -1466,7 +1466,7 @@ if __name__ == "__main__":
     MAGIC_COMMAND_FIND_VALGRIND = 'test -d ' + JENKINS_ARTIFACTS_DIR + '/valgrind/RELEASE_NAME'
     MAGIC_COMMAND_FIND_IGPROF = 'test -d ' + JENKINS_ARTIFACTS_DIR + '/igprof/RELEASE_NAME'
     MAGIC_COMMAND_FIND_PROFILING = 'test -d ' + JENKINS_ARTIFACTS_DIR + '/profiling/RELEASE_NAME'
-    MAGIC_COMMAND_FIND_PROFILING_CHECKS_FILTER1 = 'test -s ' + JENKINS_ARTIFACTS_DIR + '/profiling/RELEASE_NAME/ARCHITECTURE/WORKFLOW/step3_RAW2DIGI_L1Reco_RECO_RECOSIM_PU.resources.json && echo ARCHITECTURE/WORKFLOW/step3_RAW2DIGI_L1Reco_RECO_RECOSIM_PU.resources'
+    MAGIC_COMMAND_FIND_PROFILING_CHECKS_FILTER1 = 'test -s ' + JENKINS_ARTIFACTS_DIR + '/profiling/RELEASE_NAME/ARCHITECTURE/WORKFLOW/step3_*.resources.json && echo ARCHITECTURE/WORKFLOW/step3_*.resources'
```

Comment:
na, `ARCHITECTURE/WORKFLOW/step3_*.resources` should be the expand name. see my last comment